lms/templates/ux/reference/legacy/course-skeleton.html

```diff
@@ -0,0 +1,945 @@
+## mako
```

Comment:
That makes sense. It is particularly an issue as these legacy pages will live for a long time. I've changed it to v1.